src/components/Header.js

```diff
@@ -28,11 +30,15 @@ class Header extends PureComponent {
       callButtons
     } = this.props;
     const { showDropDownList } = this.state;
+    const hasProfileImage = profileImageUrl && profileImageUrl.trim();
 
     return (
     <div id="ch_header" className="ch-header">
       { showArrowBack && <i className="material-icons arrow-back" onClick={onBack}>arrow_back</i>}
-      { profileImageUrl && <div className="ch-header-image" style={{backgroundImage:`url(${profileImageUrl})`}}></div>}
+
+      { hasProfileImage && <div className="ch-header-image" title={profileImageAlt} style={{backgroundImage:`url(${profileImageUrl})`}}></div>}
+      { !hasProfileImage && profileImageAlt && <Avatar name={profileImageAlt} className="ch-header-image" maxInitials={2} size={40} round={true} /> }
```

Comment:
@abhishekbadoni-bigstep you can write this condition in ternary form.

Reply:
Done!